app/workers/event_signup_reminder_worker.rb

```diff
@@ -0,0 +1,27 @@
+class EventSignupReminderWorker
+  include Sidekiq::Worker
+
+  def perform(event_signup_id, time_until)
+    event_signup = EventSignup.includes(:event_users).find(event_signup_id)
+    if notifyable(event_signup, time_until)
```

Comment:
Replace the if statement with:
`notify(event_signup) if notifyable(event_signup, time_until)`